gossip/gossip.proto

```diff
@@ -33,61 +33,44 @@ option (gogoproto.goproto_unrecognized_all) = false;
 // Request is the request struct passed with the Gossip RPC.
 message Request {
   // Requesting node's ID.
-  optional int32 node_id = 1 [(gogoproto.nullable) = false,
-      (gogoproto.customname) = "NodeID",
+  int32 node_id = 1 [(gogoproto.customname) = "NodeID",
       (gogoproto.casttype) = "github.com/cockroachdb/cockroach/proto.NodeID"];
   // Address of the requesting client.
-  optional util.UnresolvedAddr addr = 2 [(gogoproto.nullable) = false];
+  util.UnresolvedAddr addr = 2 [(gogoproto.nullable) = false];
   // Local address of client on requesting node (this is a kludge to
   // allow gossip to know when client connections are dropped).
-  optional util.UnresolvedAddr l_addr = 3 [(gogoproto.nullable) = false];
+  util.UnresolvedAddr l_addr = 3 [(gogoproto.nullable) = false];
   // Maximum sequence number of gossip from this peer.
-  optional int64 max_seq = 4 [(gogoproto.nullable) = false];
-  // Reciprocal delta of new info since last gossip.
-  optional InfoStoreDelta delta = 5 [(gogoproto.nullable) = false];
+  int64 max_seq = 4;
+  // Delta of new Infos since last gossip.
+  map<string, Info> infos = 5;
```

Comment:
I think you should keep this variable named "delta". It's so much more descriptive.